### ISSUE TITLE
--host,--port and others are not working

### ISSUE THREAD

**Issue by Programmeryusuf** [codeigniter4/CodeIgniter4#2329]
--host ,--port and others are not working

**Comment by lonnieezell** [comment]
You know - there was a nice bit of instruction in the template that basically asked for details - well, we really need those details to help you. :) 

I use --port all of the time and it works fine for me. To be honest I haven't tried --host recently. 

What were you trying (exactly), what happened, etc. What PHP version, OS, etc.

**Comment by lonnieezell** [comment]
I just ran some tests and they all work for me on Mac OSX, running PHP 7.2.

- modified `/etc/hosts` and added `127.0.0.1 example.local`
- ran `php spark serve --host example.local`
- visited web page. It worked as expected. 
- ran `php spark serve --port 8083`
- visited `localhost:8083`. It worked as expected 

I did not try the PHP version because I'm lazy and didn't feel like tracking down a path. :) However, these are all just passing variables to PHP's built in web server, so I would expect it to work.
